apps/comments-ui/src/App.tsx

```diff
@@ -224,7 +224,7 @@ const App: React.FC<AppProps> = ({scriptTag}) => {
             <CommentsFrame ref={iframeRef}>
                 <ContentBox done={done} />
             </CommentsFrame>
-            {state.comments.length > 0 ? <AuthFrame adminUrl={options.adminUrl} onLoad={initAdminAuth}/> : null}
+            {state.initStatus === 'success' && <AuthFrame adminUrl={options.adminUrl} onLoad={initAdminAuth}/>}
```

Comment:
Maybe you could reuse the `done` constant to avoid logic duplication, what do you think?

Reply:
👍  thanks for the suggestion, I've made that change